sdk/storage/azure-storage-file-share/tests/conftest.py

```diff
@@ -1,33 +1,37 @@
-# --------------------------------------------------------------------------
-#
+# -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-#
-# The MIT License (MIT)
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the ""Software""), to
-# deal in the Software without restriction, including without limitation the
-# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
-# sell copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
-# IN THE SOFTWARE.
-#
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
 # --------------------------------------------------------------------------
-import sys
 
-# fixture needs to be visible from conftest
+import os
+import platform
+
+import pytest
+
+from devtools_testutils import (
+    add_general_regex_sanitizer,
+    add_header_regex_sanitizer,
+    add_oauth_response_sanitizer,
+    add_uri_string_sanitizer,
+    test_proxy
+)
 
-# Ignore async tests for Python < 3.5
+# Ignore async tests for PyPy
 collect_ignore_glob = []
-if sys.version_info < (3, 5):
```

Comment:
When setting up `conftest.py` I just "borrowed" 😛 the `conftest.py` from datalake. Should I pop back in this explicit version check? Tests seemed to run fine in _this_ file without it, but want to point it out 😄